casper/src/main/rholang/BasicWalletFaucet.rho

```diff
@@ -0,0 +1,17 @@
+//scalapackage coop.rchain.rholang.mint
+
+//requires MakeMint, BasicWallet
+
+contract @"BasicWalletFaucet"(@mint, return) = {
+  new this in {
+    contract this(@amount, @algorithm, @publicKey, return) = {
+      new walletPurseCh in {
+        @(mint, "makePurse")!(amount, *walletPurseCh) |
+        for(@walletPurse <- walletPurseCh) {
+          @"BasicWallet"!(walletPurse, algorithm, publicKey, *return)
+        }
+      }
+    } |
+    return!(*this)
+  }
+}
```

Comment:
I wonder if allowing for unlimited REV to go around in testnet is a good idea. I know other blockchain testnets do not allow this but maybe that is just a side-effect of PoW.

Reply:
I don't like it either but I hope that this idea has been attacked and considered by "big heads" from different perspectives.

Reply:
What would be the argument against unlimited REV? We want people to be able to try out the platform, so there needs to be some way of getting REV. If we went with a faucet that had a limited supply then the number of people able to use test net also becomes limited. Or if there was unlimited REV but their was some barrier to obtaining it (e.g sending an application to the coop) then I think people would wonder why this barrier was put in place.